src/controllers/user.controllers.js

```diff
@@ -74,6 +89,22 @@ export const passwdEdit = async (data, id) => {
   return userUpdated(data, id);
 };
 
+export const userDeletion = async (id) => {
+  const userExist = await User.findOne({ _id: id.id });
+
+  if (!userExist || userExist.statusUser === 0) return { status: 'invalid', message: 'User not found' };
+
+  await User.findOneAndUpdate(
+    {
+      _id: id.id,
+    },
+    { statusUser: 0 },
+    { new: true }
+  );
+
+  return 'The account has been deleted';
```

Comment:
Might not function well on the front-end as the structure of this response differs from the structure of the rest of responses.